ui-tests/src/test/java/io/syndesis/qe/hooks/OnFailHooks.java

```diff
@@ -0,0 +1,108 @@
+                log.error("######## Open Issues for scenario: " + tag);
+                filter.put("labels", tag);
+                //for some reason this request returns only open issues - probably default github setting
+                List<Issue> reportedIssues = issueService.getIssues(repositoryService.getRepository("syndesisio", "syndesis"), filter);
+                filter.put("state", "closed");
+                List<Issue> closedIssues = issueService.getIssues(repositoryService.getRepository("syndesisio", "syndesis"), filter);
+                if (reportedIssues.size() == 0) {
+                    scenario.embed("#### No previously reported issues found.".getBytes(), "text/plain");
+                    log.error("#### No previously reported issues found.");
+                    continue;
+                }
+
+                //print all open issues, if closed save to print later so it is sorted open/closed
+                for (Issue knownIssue : reportedIssues) {
+                    if (knownIssue.getState().equalsIgnoreCase("open")) {
+                        scenario.embed(("#### Title: " + knownIssue.getTitle()).getBytes(), "text/plain");
+                        log.error("#### Title: " + knownIssue.getTitle());
+                        scenario.embed(("#### Link: " + knownIssue.getHtmlUrl()).getBytes(), "text/plain");
+                        log.error("#### Link: " + knownIssue.getHtmlUrl());
+                        scenario.embed("#### ------------------------------------------------------------------".getBytes(), "text/plain");
+                        log.error("#### ------------------------------------------------------------------");
+                    } else {
+                        closedIssues.add(knownIssue);
+                    }
+                }
+                if (closedIssues.size() == 0) {
+                    continue;
+                }
+
+                scenario.embed(("######## Resolved issues for tag: " + tag).getBytes(), "text/plain");
+                log.error("######## Resolved issues for tag: " + tag);
+                //print closed issues
+                for (Issue closedIssue : closedIssues) {
+                    scenario.embed(("#### Title: " + closedIssue.getTitle()).getBytes(), "text/plain");
+                    log.error("#### Title: " + closedIssue.getTitle());
+                    scenario.embed(("#### Link: " + closedIssue.getHtmlUrl()).getBytes(), "text/plain");
+                    log.error("#### Link: " + closedIssue.getHtmlUrl());
+                    scenario.embed("----------------------------------------------------------------------".getBytes(), "text/plain");
+                    log.error("----------------------------------------------------------------------");
+                }
+            }
+        } catch (IOException e) {
+            log.error("Error while processing GitHub issues, lets spam twitter of Satya Nadella");
```

Comment:
I'd vote to change this log and following statement to log.error("Error while processing GitHub issues", e");